deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/QuarkusJavaClientCodegen.java

```diff
@@ -121,14 +121,50 @@ public void postProcess() {
     }
 
     @Override
-    protected String getSymbolName(String input) {
-        for (Entry<String, String> entry : specialCharReplacements.entrySet()) {
-            if (input.startsWith(entry.getKey())) {
-                return input.length() > entry.getKey().length()
-                        ? entry.getValue() + "_" + input.substring(entry.getKey().length())
-                        : entry.getValue() + "_symbol";
+    public String toEnumVarName(String value, String datatype) {
+        String enumValue = value;
+        if (enumValue.isEmpty()) {
+            return "EMPTY";
+        }
+
+        if (getSymbolName(enumValue) != null) {
+            return getSymbolName(enumValue).toUpperCase(Locale.ROOT);
+        }
+
+        if (" ".equals(enumValue)) {
+            return "SPACE";
+        }
+
+        if ("Integer".equals(datatype) || "Long".equals(datatype) ||
+                "Float".equals(datatype) || "Double".equals(datatype) || "BigDecimal".equals(datatype)) {
+            String varName = "NUMBER_" + enumValue;
+            varName = varName.replaceAll("-", "MINUS_");
+            varName = varName.replaceAll("\\+", "PLUS_");
+            varName = varName.replaceAll("\\.", "_DOT_");
```

Comment:
We don't need to use regex here, right?

```suggestion
            varName = varName.replace("-", "MINUS_");
            varName = varName.replace("+", "PLUS_");
            varName = varName.replace(".", "_DOT_");
```

Reply:
Hi @hbelmiro, TY!

I copied it from super, I will try to remove it in the next commits.